app/main.py

```diff
@@ -395,6 +395,19 @@ def setup_env(request):
         # support UTF-8 but don't render UTF-8 symbols such as u'\xab'.
         env.back_chevron = u'<<'
 
+    env.enable_maps = (
+        env.enable_javascript
+        and not env.config.zero_rating_mode
+        and env.maps_api_key)
+    env.enable_analytics = (
```

Comment:
I don't have full context here, but external api calls are totally prohibited with zero-rating mode, or is this to save bandwidth?